src/coverlet.core/Helpers/FileSystem.cs

```diff
@@ -3,11 +3,46 @@
 
 namespace Coverlet.Core.Helpers
 {
-    internal class FileSystem : IFileSystem
+    public class FileSystem : IFileSystem
     {
         public bool Exists(string path)
         {
             return File.Exists(path);
         }
+
+        public void WriteAllText(string path, string contents)
+        {
+            File.WriteAllText(path, contents);
+        }
+
+        public string ReadAllText(string path)
+        {
+            return File.ReadAllText(path);
+        }
+
+        public FileStream OpenRead(string path)
```

Comment:
Should return `Stream` so we can mock it.

Reply:
Done